crates/codegen/ebnf/src/builder.rs

```diff
@@ -11,6 +11,21 @@ use inflector::Inflector;
 
 use crate::model::{Definition, DefinitionKind, Entry, Expression, Value};
 
+#[allow(dead_code)]
+#[derive(strum_macros::AsRefStr)]
+#[strum(serialize_all = "snake_case")]
+enum BuiltInLabel {
```

Comment:
> I copied the BuiltInLabels enum from the grammar generator crate because it was private and not directly accessible from the ebnf crate. But I think it may make sense to move the enum into the language_definition crate? It may make it possible to remove all the duplication.

Sounds good. Please feel free to move it and update all callers, or we can do it in a separate PR as well.